Assets/PatchKit/UnityEditorExtension/Editor/UI/LinkAppScreen.cs

```diff
@@ -42,87 +42,131 @@ public override void UpdateIfActive()
 
     public override void Draw()
     {
-        GUILayout.Label(
-            string.Format(
-                "Link your PatchKit application for {0}.",
-                _platform.ToDisplayString()),
-            EditorStyles.boldLabel);
+        EditorGUILayout.BeginHorizontal();
+        {
+            if (GUILayout.Button(
+                new GUIContent(_arrowIcon, "Change application"),
+                GUILayout.Width(35),
+                GUILayout.Height(20)))
+            {
+                Dispatch(() => Cancel());
+            }
+            
+            GUILayout.Space(220);
+            if (GUILayout.Button("Create new app", GUILayout.Width(130), GUILayout.Height(20)))
+            {
+                Dispatch(() => CreateNew());
+            }
+        }
+        EditorGUILayout.EndHorizontal();
+        EditorGUILayout.BeginHorizontal();
+        {
+            GUILayout.FlexibleSpace();
+            GUILayout.Label(
+                string.Format(
+                    "Link your PatchKit application for {0}.",
+                    _platform.ToDisplayString()),
+                EditorStyles.boldLabel);
+            GUILayout.FlexibleSpace();
+        }
+        EditorGUILayout.EndHorizontal();
 
+        ShouldFilterByPlatform = EditorGUILayout.Toggle("Filter apps by platform", ShouldFilterByPlatform);
         EditorGUILayout.Space();
 
         _scrollViewVector = EditorGUILayout.BeginScrollView(
             _scrollViewVector,
             EditorStyles.helpBox);
 
-        foreach (App app in Apps)
-        {
-            DrawApp(app);
-        }
-
-        EditorGUILayout.EndScrollView();
-
-        EditorGUILayout.Space();
 
-        if (GUILayout.Button("Create new app", GUILayout.ExpandWidth(true)))
+        for (int i = 0; i < Apps.Length; i++)
         {
-            Dispatch(() => CreateNew());
+            App app = Apps[i];
+            DrawApp(app, i % 2);
         }
 
-        EditorGUILayout.Space();
-
-        if (GUILayout.Button("Cancel", GUILayout.ExpandWidth(true)))
-        {
-            Dispatch(() => Cancel());
-        }
+        EditorGUILayout.EndScrollView();
     }
 
-    private void DrawApp(App app)
+    private void DrawApp(App app, int i)

```

Comment:
What is this `i` argument for?

If you want to make different background colors for odd & even rows, please create argument named `isEven` or `isOdd`.